apps/test/unit/templates/rubrics/RubricSettingsTest.jsx

```diff
@@ -29,10 +46,14 @@ describe('RubricSettings', () => {
   });
 
   afterEach(() => {
-    fetchStub.restore();
-    utils.queryParams.restore();
+    if (clock) {
+      clock.restore();
```

Comment:
Not restoring the use of the fake timers made the React Testing Library's waiting not work since it uses the timers internally to poll for changes. If one of my tests ran after this one, it would timeout.